src/Analysis.cpp

```diff
@@ -452,10 +452,12 @@ void CodeAnalyzer::doAnalysis(Function &F, std::string optLoadFileName) {
 
         loadFunctions();
 
-        TempFileManager astTempFile("astTempFile");
-        tempfile_util::dumpASTIntoTempFile(optLoadFileName, astTempFile);
+        TempFileManager astInfoTempFile("astInfoTempFile");
+        tempfile_util::dumpASTInfoIntoTempFile(optLoadFileName, astInfoTempFile);
 
-        auto annotations = getAnnotationStrings(astTempFile);
+        TempFileManager annotationsTempFile = TempFileManager("annotationsTempFile");
+
+        auto annotations = getAnnotationStrings(astInfoTempFile); // TODO! FIX THIS
```

Comment:
What's wrong with this? The "TODO! FIX THIS" seems bad - but I don't see the issue looking at this locally.

Reply:
Comments not supposed to be there; deleting. It's all fine.